src/main/java/io/github/thebusybiscuit/slimefun4/core/guide/options/GuideLayoutOption.java

```diff
@@ -1,115 +0,0 @@
-package io.github.thebusybiscuit.slimefun4.core.guide.options;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Optional;
-
-import org.bukkit.ChatColor;
-import org.bukkit.Material;
-import org.bukkit.NamespacedKey;
-import org.bukkit.entity.Player;
-import org.bukkit.inventory.ItemStack;
-import org.bukkit.inventory.meta.ItemMeta;
-
-import io.github.thebusybiscuit.slimefun4.api.SlimefunAddon;
-import io.github.thebusybiscuit.slimefun4.core.guide.SlimefunGuide;
-import io.github.thebusybiscuit.slimefun4.core.guide.SlimefunGuideLayout;
-import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
-import io.github.thebusybiscuit.slimefun4.utils.ChatUtils;
-import io.github.thebusybiscuit.slimefun4.utils.SlimefunUtils;
-
-class GuideLayoutOption implements SlimefunGuideOption<SlimefunGuideLayout> {
-
-    @Override
-    public SlimefunAddon getAddon() {
-        return SlimefunPlugin.instance();
-    }
-
-    @Override
-    public NamespacedKey getKey() {
-        return new NamespacedKey(SlimefunPlugin.instance(), "guide_layout");
-    }
```

Comment:
Removing this is a terrible idea... This way you effectively also removed the Cheat Sheet...